CONTRIBUTING.md

```diff
@@ -156,11 +157,47 @@ You can update your [fork](https://help.github.com/articles/about-forks/) of the
 GitHub has a [nice introduction](https://help.github.com/articles/github-flow/) to the pull request workflow, but please [get in touch](#get-in-touch) if you have any questions.
 
 ## Example pull request
-<img align="right" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
+<img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>  
 
-<br />
-<br />
 
+## Fixing Travis Remark errors
+
+We use a linter called [Remarkjs](https://github.com/remarkjs/remark-lint) to ensure all of our specification documentation is consistent.
+We have found this causes typical errors that are flagged by our continous integration service called [Travis CI](https://travis-ci.org/).
+This behavior is commonly seen when a PR is opened and Travis returns an error. 
+To fix this error please follow this process:
+
+#### 1. Download Remark-CLI and our style guide
+
+Remark-CLI can be downloaded with the help of [npm](https://www.npmjs.com/). 
+Once npm has been downloaded, please follow the [installation instructions on Remark's GitHub.](https://github.com/remarkjs/remark/tree/master/packages/remark-cli#install)
+
+We use a particular style guide for formatting. 
+To get this style guide please run: 
+
+```
+npm install remark-cli@5.0.0 remark-lint@6.0.2 remark-preset-lint-recommended@3.0.2 remark-preset-lint-markdown-style-guide@2.1.2
+```
```

Comment:
perhaps packaging these up? a potential of this would be needing to maintain that package too. This was grabbed from our https://github.com/bids-standard/bids-specification/blob/master/.travis.yml config line 8. That is the place I know we check our formatting

Reply:
I just meant something equivalent to a `requirements.txt` in Python. Here's something that could work:

`npm-requirements.txt`:
```
remark-cli@5.0.0
remark-lint@6.0.2
remark-preset-lint-recommended@3.0.2
remark-preset-lint-markdown-style-guide@2.1.2
```

Then it could be installed locally or on Travis with:

```Shell
cat npm-requirements.txt | xargs npm install
```

But there might be something equivalent in the NodeJS world. A very cursory glance didn't find it.

Reply:
yeah we can do that - I think definitely locally and can be extended to Travis. Will ease the process. have added this

Reply:
I also looked for a `pip install -r requirements.txt` equivalent in the `npm` world and it seems like it doesn't exist. 

>  Is this the only place these versions are recorded, or do we need to keep this in sync with some other document

needs to stay in sync with CircleCI [config.yml](https://github.com/bids-standard/bids-specification/blob/master/.circleci/config.yml), so I suggest that in the CircleCI config, we also use the newly created `npm-requirements.txt` file from this PR. Can you do that @franklin-feingold ?

Also, perhaps we can add a sentence for Windows users, who don't have command line tools like `cat`, `|`, or `xargs` by default: These people should open the `npm-requirements.txt` file and install each line by hand, as a practical solution

Reply:
At least with PowerShell, people can do:

```Powershell
npm install @(cat npm-requirements.txt)
```

In bourne-like shells, we could also have people do:

```Bash
npm install $(cat npm-requirements.txt)
```

There's probably some unholy utterance for `cmd.exe`, but I doubt many people using `npm` on Windows won't have access to Bash or Powershell.

Reply:
oh yeah thanks I do check remark with circle - added in npm-requirements to circle and the Windows cases Chris identified